docker-compose-indexer.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+
+services:
+  indexer:
+    container_name: "${INDEXER_CONTAINER:-algorand-sandbox-indexer}"
+    build:
+      context: .
+      dockerfile: ./images/indexer/Dockerfile
+      args:
+        URL: "${INDEXER_URL}"
+        BRANCH: "${INDEXER_BRANCH}"
+        SHA: "${INDEXER_SHA}"
+    ports:
+      - ${INDEXER_PORT:-8980}:8980
+    restart: unless-stopped
+    environment:
+      PORT: "8980"
+      SNAPSHOT: ""
+      CONNECTION_STRING: "host=indexer-db port=5432 user=algorand password=algorand dbname=indexer_db sslmode=disable"
+      ALGOD_ADDR: "follower:4003"
```

Comment:
I believe this would be `algod:4003` since the follower node is running in the algod container